src/dmd/mars.d

```diff
@@ -287,6 +287,15 @@ private int tryMain(size_t argc, const(char)** argv, ref Param params)
         return global.errors ? EXIT_FAILURE : EXIT_SUCCESS;
     }
 
+    /*
+    Print a message to make it clear when warnings are treated as errors.
+    */
+    static void errorOnWarning()
+    {
+        error(Loc.initial, "warnings are treated as errors");
```

Comment:
You can just use inform() instead of error().

Reply:
I don't think this makes sense, as `error()` gives a nice red `Error:` prefix